home/import_assessments.py

```diff
@@ -96,9 +104,31 @@ def save_assessments(self) -> None:
             )
 
     def parse_file(self) -> list["AssessmentRow"]:
+        """
+        Converts the iterator to a list of rows.
+        If there are rows:
+            a. Extracts the original headers from the first row.
+            b. Converts the original headers to snake_case and creates a mapping from original headers to snake_case headers.
+            c. Validates that the snake_case headers contain all mandatory headers.
+            d. Transforms each row to use snake_case headers.
```

Comment:
The snake_case transformation probably belongs in the shared code, so all importers get cleaned-up keys. That would also avoid having to juggle the mapping from old to new keys in here.


Can we do that without breaking anything in the other importers?

Reply:
The snake_case even though it's in helpers.py is not shared otherwise it will break contentset importer which looks up headers in pascal case. if we needed to integrate it in the generic import process then we would first need to fix the code in content pages.

Reply:
Ah. In that case, can you add a `TODO` comment to move this to the shared code once we're able to work on the contentpage import without interfering with other ongoing work?